package/obs/rmt-server.changes

```diff
@@ -5,8 +5,10 @@ Tue Jan 21 10:49:54 UTC 2025 - Jesús Bermúdez Velázquez <jesus.bv@suse.com>
   * rmt-server-pubcloud:
     * Add pubcloud_reg_code column to systems table
     * Add cache directories and their expiration times to the scrubber
+  * Update Puma webserver
```

Comment:
did we?

Reply:
Yes, but forgot to add it to changelog: https://github.com/SUSE/rmt/pull/1312/files